### DISCUSSION TITLE
Test failure

### DISCUSSION THREAD

**Discussion by maccroc** [runelite/runelite#16631] [Q&A]
This one specifically: 
runelite\runelite-client\target\surefire-reports\TEST-net.runelite.client.plugins.chatcommands.ChatCommandsPluginTest.xml

With the error message;

There are test failures.
--------------------------
Please refer to C:\Users\x\IdeaProjects\runelite\runelite-client\target\surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
------------------------------
And from what i can see in the repo, it was updated yesterday.

Anyone who has a fix or can tell me if this is trying to get resolved as i am writing this?

**Comment by Nightfirecat** [comment]
That message doesn't have any information in it. You can see from CI on the repository that we don't have any test failures in the codebase right now, so this likely isn't an issue with the tests in general. I recommend you either [join our Discord server](https://runelite.net/discord) and ask for help in the `#runelite` channel, or post your full build log here.
